controller_manager/src/controller_manager/controller_manager_interface.py

```diff
@@ -108,13 +102,13 @@ def start_stop_controllers(names, st):
     resp = s.call(SwitchControllerRequest(start, stop, strictness))
     if resp.ok == 1:
         if st:
-            print("Started {} successfully".format(names))
```

Comment:
I think your PR needs rebasing. Please don't change the prints.

Reply:
Why should the spawner not be responsible for shutting down all running controllers?  I don't see a problem with making that change, I'm just trying to understand it.

And I'm not sure how the prints ended up being changed, it must have been my text editor.  I'll fix those.